examples/cae/README.md

```diff
@@ -0,0 +1,245 @@
+   ./ghpc create community/examples/cae-slurm.yaml -w --vars project_id=<project>
+   ```
+
+1. Deploy the `setup` group
+
+   Call the following ghpc command to deploy the cae-slurm blueprint.
+
+   ```bash
+   ./ghpc deploy cae-slurm
+   ```
+
+   The next `ghpc` prompt will ask you to **display**, **apply**, **stop**, or
+   **continue** without applying the `setup` group. Select 'apply'.
+
+   This group will create a network and file systems to be used by the cluster.
+
+   > [!WARNING]
+   > This ghpc command will run through 2 deployment groups (3 if you populate
+   > & activate the `software_installation` stage) and prompt you to apply each one.
+   > If the command is cancelled or exited by accident before finishing, it can
+   > be rerun to continue deploying the blueprint.
+
+1. Deploy the `software_installation` group (_optional_).
+
+   > [!NOTE]
+   > Installation processes differ between applications. Some come as a
+   > precompiled binary with all dependencies included, others may need to
+   > be built from source, while others can be deployed through package
+   > managers such as spack. This deployment group is intended to be used
+   > if the software installation process requires substantial amount of time (e.g.
+   > compilation from source). By building the software in a separate
+   > deployment group, this process can be done before the cluster is
+   > up, minimizing costs.
+   >
+   > [!NOTE]
+   > By default, this deployment group is disabled in the reference design. See
+   > [Software Installation Patterns](#software-installation-patterns) for more information.
+
+   If this deployment group is used (needs to be uncomment in the blueprint first),
+   you can return to the ghpc command which will ask you to **display**, **apply**,
+   **stop**, or **continue** without applying the `software_installation` group.
+   Select 'apply'.
+
+1. Deploy the `cluster` group
+
+   The next `ghpc` prompt will ask you to **display**, **apply**, **stop**, or
+   **continue** without applying the `cluster` group. Select 'apply'.
+
+   This deployment group contains the Slurm cluster and with compute partitions
+   and a partition for Chrome remote desktop visualization nodes.
+
+1. Set up Chrome Remote Desktop
+
+   One or multiple Chrome Remote Desktop (CRD) sessions can be started dynamically
+   through Slurm.
+
+   - Follow
+     [the instructions](../README.md#hpc-slurm-chromedesktopyaml--)
+     for setting up the Remote Desktop.
+
+## Teardown Instructions
+
+> [!NOTE]
+> If you created a new project for testing of the CAE solution, the easiest way to
+> eliminate billing is to delete the project.
+
+When you would like to tear down the deployment, each stage must be destroyed.
+Since the `software_installation` and `cluster` depend on the network deployed
+in the `setup` stage, they must be destroyed first. You can use the following
+commands to destroy the deployment.
+
+```bash
+./ghpc destroy cae-slurm --auto-approve
```

Comment:
I would recommend not adding the --auto-approve here as it will not ask for confirmation or steps. It would be safer let the user interact. What do you think?

Reply:
Good point. I agree. I modified this part of the README accordingly